### PR TITLE
UPDATED POLICY FOR GIT USAGE

### PR DESCRIPTION
**Pull request by James-8212913**
This offers a standard way to name and manage branches, write commit messages and construct pull requests. Updated text only ready for proofing before being pushed to production on the master

### Completes [name of link](insert weblink address - github page)

--------------
# The issue

What was the issue?

# The solution

What did you do to fix it?

# Visuals

Before | After
----- | -----
A | B

# Test cases

How can we thoroughly test this? Include sign-in information, if needed.

### CODE REVIEW

**Comment by James-8212913** [comment]
Issue

The GIT procedures for the group are still being developed.

Solution

This is offers a way for the group to collaborate on GIT in a more standard manner covering branching/ commit messages/ pull request format.

Visuals N/A

Test Case

It needs proofing before being pushed to production

File: source/assessment_criteria.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Both R and Python are Objective languages in the way they process Data, this mea
 
 ### Graphics and Visualisations
 
-R has got is on Python in this instance. The visualisations possible with R are of a higher quality generally than those from Python for a similar amount of code. In saying this, Python still offers a significant number of options and the graphics produced are of production quality. If the product was to be progressed further the use of Plotly could be integrated to ensure the visualisations were interactive and more user friendly. 
+R has got is on Python in this instance. The visualisations possible with R are of a higher quality generally than those from Python for a similar amount of code. In saying this, Python still offers a significant number of options and the graphics produced are of production quality. If the product was to be progressed further the use of Plotly could be integrated to ensure the visualisations were interactive and more user friendly.
```
Comment (Yun-Oh) [inline source/assessment_criteria.md:30]: Our main reasoning on choosing Python was to improve the python knowledge as this was not taught in any capstone subjects.  Also, advantage as a general programming languages e.g. sending API request to yahoo to retrieve the share price information

**Review by Yun-Oh — APPROVED** [review]
Thank you for updating the assessment criteria file and the git procedure.

Our main reasoning on choosing Python was to improve the python knowledge as this was not taught in any capstone subjects.  Also, advantage as a general programming languages e.g. sending API request to yahoo to

**Comment by pri-mdsi** [comment]
@James-8212913 the git procedure looks good. Do we really need A/B testing though? As the project in mainly focussed on EDA and creating visualisations

**Pull request merged**